content/blog/2022-05-03-e2e-cv-pt1.md

```diff
@@ -0,0 +1,307 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 1: Data Versioning and ML pipelines'
```

Comment:
```suggestion
  'End-to-End Computer Vision API, Part 1: Data Versioning and ML Pipelines'
```

Reply:
But maybe move `Data Versioning and ML Pipelines` to the `description(Long)` fields? Not sure, but it feels like a subtitle. This one is completely up to you, just a thought

Reply:
I'd like to keep `Data Versioning and ML Pipelines` in the title so that it's clear from it what the post will be about.